tests/foreign_object/models/customers.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
Is it possible to reuse some existing models (without too much complication). We try to keep the number of models minimized and each additional model increases the run time of the test suite.